CONTRIBUTING.md

```diff
@@ -1,5 +1,31 @@
-Syntax coding style guidelines
-==============================
+# How to report issues
+
+Before reporting an issue with GitHub, be sure that:
+* you are using the git version of radare2
+* you are using a clean installation
```

Comment:
@radare what are the commands to be sure that you have everything clean? I'd like to add those in this line.